rest/data/project_test.go

```diff
@@ -283,3 +284,92 @@ func (s *ProjectConnectorGetSuite) TestFetchKeyOutOfBoundDesc() {
 	s.NoError(err)
 	s.Len(projects, 0)
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// Tests project create action
+type ProjectConnectorCreateUpdateSuite struct {
+	ctx      Connector
```

Comment:
I'd avoid calling this ctx to avoid confusion with context.Context

Reply:
Oh, agree, copied it from somewhere...